### DISCUSSION TITLE
Multi-timbral performance patch

### DISCUSSION THREAD

**Discussion by Lonolo** [probonopd/MiniDexed#774] [Ideas]
Does a performance patch already exist in the build that assigns all 8 DX7 instances to discrete MIDI channels?

For example:
* MIDI CH1: BASS 1
* MIDI CH2: E. PIANO 2
* MIDI CH3: STRINGS 4
* ...
* MIDI CH8: SYNLEAD 2

Such a setup could act as a flexible starting point and template for users aiming to create multi-timbral performances efficiently.

If not, is there a better approach to building a multi-channel patch?

**Comment by Banana71** [comment]
I can't find the post with my templates anymore, but there is a performance bank with templates on my site.
The "128_Laboratory" folder contains templates for creating performances and now also the MultiTimbral.ini again.

https://github.com/Banana71/Soundplantage


_I found the old post after all. :https://github.com/probonopd/MiniDexed/discussions/325#discussioncomment-5498474
Since then, however, a lot has changed in the structure of miniDexed and the performances stored there no longer work without further ado.
This is only useful for history lessons :-)_

**Reply by Lonolo** [reply]
Thanks, those are what I'm referring to - the Multitimbral and DrumMap ini's!

Not sure why those, or updated versions aren't included in the build already?

Having to normalize the detune, panning and volume levels etc. from any of the existing performance patches to go multi-timbral gets frustrating.

**Reply by Banana71** [reply]
> Not sure why those, or updated versions aren't included in the build already?

Only the "performance" folder and the "miniDexed Performances.pdf" file will be integrated into the miniDexed build.
If you want, I could move the template folder "128_Labaratory" into the performance folder!?
Then they will be integrated in the next miniDexed update!?

But I would like the OK from @probonopd for that.

**Reply by blixabloxa** [reply]
Yes, I've already done that manually with your Laboratory patches. Thanks for your contributions Banana71, your patches really make Minidexed shine. 👍😁

**Reply by probonopd** [reply]
Sure, why not. Let's do that 👍

**Reply by Banana71** [reply]
job done 🧪🔬⚗️🧫

**Reply by probonopd** [reply]
It's in the latest build. Thanks @Banana71

**Comment by Lonolo** [comment]
That would be great! 
... & of course if @probonopd agrees.
